doc/user/content/transform-data/patterns/time-travel-queries.md

```diff
@@ -0,0 +1,153 @@
+the specified time period.
+
+* **For sources, tables and materialized views:** increasing the history retention
+  period will not restore older historical data that was already outside the
+  previous history retention period before the change.
+
+* **For indexes:** if all of the underlying source, table, and materialized view
+  historical data is available for the increased history retention period, the
+  index can use that data to backfill as far back as the underlying historical
+  data is available.
+
+#### Decreasing the history retention period
+
+When you decrease the history retention period for an object:
+
+* Newly produced historical data is retained for the new, shorter history
+  retention period.
+
+* Historical data outside the new, shorter history retention period is no longer
+  retained. If you subsequently increase the history retention period again,
+  the older historical data may already be unavailable.
+
+### Resource utilization
+
+It's important to note that increasing the history retention period for an
+object will lead to increased resource utilization in Materialize.
+
+* **For sources, tables and materialized views:** increasing the history
+    retention period for these objects increases the amount of historical data
+    that is retained in the storage layer. You can expect storage resource
+    utilization to increase, which may incur additional costs.
+
+* **For indexes:** increasing the history retention period for an index
+    increases the amount of historical data that is retained in memory in the
+    cluster maintaining the index. You can expect memory resource utilization
+    for that cluster to go up, which might require you to size up your cluster
+    and consume additional compute credits.
+
+### Clean-up
+
+The history retention period represents the minimum amount of historical data
+guaranteed to be retained by Materialize. History clean-up is processed in the
+background, so older history may be accessible for the period of time between
+when it falls outside the retention period and when it is cleaned up.
+
+### Best practices
+
+Given the additional memory costs associated with increasing the history
+retention period on indexes, if you don't need the performance benefits of an
+index - in particular for the use case of [durable subscriptions]((/sql/subscribe#durable-subscriptions))
+- you should consider creating a materialized view for your subscription query
+and configure the history retention period on that materialized view
+instead.
+
+Similarly, because of the increased storage costs and processing time for the
+additional historical data on whichever object has an increased history
+retention period, you should consider configuring history retention period on
+the index or materialized view directly powering the subscription, rather than
+all the way through the dependency chain from the source to the index or
+materialized view.
+
+## Examples
```

Comment:
Thoughts from a second pass: since we're spinning this as a pattern, it feels like the how-to for point-in-time queries and durable subscriptions should live here, rather than being buried in the reference documentation. Also, we need to bring `AS OF` back to the [`SELECT`](https://materialize.com/docs/sql/select/) reference page.

Eventually, I think we'll want to extricate the SQL reference bits as a shortcode that can be applied to the low-level reference pages, and focus this page on just guidance on how to implement the time traveling patterns in practice.

Reply:
Concur.

Reply:
Just pushed up the change to move the main durable subscription guide to the time travel docs, and put a todo for point in time queries docs.